chandra_aca/star_probs.py

```diff
@@ -598,20 +607,21 @@ def model_acq_success_prob(mag, warm_frac, color=0, halfwidth=120):
 
 def sota_model_acq_prob(mag, warm_frac, color=0, halfwidth=120):
     """
-    Calculate raw SOTA model probability of acquisition success for a star with ``mag``
-    magnitude and a CCD warm fraction ``warm_frac``.  This is not typically used directly
-    since it does not account for star properties like spoiled or color=0.7.
+    Calculate raw SOTA model probability of acquisition success.
+
+    This is for a star with ``mag`` magnitude and a CCD warm fraction ``warm_frac``.
+    This is not typically used directly since it does not account for star properties
+    like spoiled or color=0.7.
 
     Uses the empirical relation::
 
        P_fail_probit = offset(mag) + scale(mag) * warm_frac + box_delta(halfwidth)
-       P_acq_fail = Normal_CDF()
-       P_acq_success = 1 - P_acq_fail
+       P_acq_fail = Normal_CDF() P_acq_success = 1 - P_acq_fail
```

Comment:
It is just in the docstring but this looks less readable to me.

Reply:
Will fix, that is VS code unexpectedly word-wrapping the entire docstring.

Reply:
Fixed.